keps/sig-network/20180612-ipv4-ipv6-dual-stack.md

```diff
@@ -893,6 +894,48 @@ of the IP family of the service. A single service which is dual-stack may use
 the same NodePort on both families.  `kube-proxy` will ensure that traffic is
 routed according to the families assigned for services.
 
+
+##### Spec.loadBalancerIP
+
+The `spec.loadBalancerIP` is a string and only allow one load-balancer
+address to be specified. A new multi value `spec.loadBalancerIPs` is added;
+
+```yaml
+apiVersion: v1
```

Comment:
This example needs a bit more work. If it is `PrefereDualStack` on a dual stack cluster then it will have both ClusterIP and ClusterIPs set. This is important for LBs that forward LB->ServiceIP or LB-IP.

Reply:
I can't see the connection between clusterIP(s) and load-balancer-ip. No extern load-balancer should balance to clusterIPs either to the node-port or to `status.loadBalancerIP.ingress`.